### PR TITLE
Finished making a basic melody generator.

### PR DESCRIPTION
**Pull request by JoachimRayski**

### CODE REVIEW

**Comment by JoachimRayski** [comment]
@gizzmo123456 check this out my dude

File: MelodyGenerator.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+class MusicGenerator:
+    filename = "song.wav"
+    wav_write = wave.open(filename, 'w')
+    wav_write.setparams((1, 2, 44100, 0, 'NONE', 'not compressed'))
+
+    values = []
+    SAMPLE_RATE = wav_write.getframerate()
+
+    D_MINOR = {"D": 587.33, "E": 659.25, "F": 698.46, "G": 783.99, "A": 880.00,
```
Comment (gizzmo123456) [inline MelodyGenerator.py]: This all very good. D_MINOR? shouldent it be called notes or frequencies as it's got more than just d_minor. Same for cords below.

File: MelodyGenerator.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+               "Bflat": 932.33, "C": 1046.50}
+    D_MINOR_CHORD_A = [D_MINOR["A"], D_MINOR["F"], D_MINOR["D"]]
+    D_MINOR_CHORD_B = [D_MINOR["Bflat"], D_MINOR["G"], D_MINOR["E"]]
+    D_MINOR_CHORD_C = [D_MINOR["C"], D_MINOR["A"], D_MINOR["F"]]
+    D_MINOR_CHORD_D = [D_MINOR["D"], D_MINOR["Bflat"], D_MINOR["G"]]
+    D_MINOR_CHORD_E = [D_MINOR["E"], D_MINOR["C"], D_MINOR["A"]]
+    D_MINOR_CHORD_F = [D_MINOR["F"], D_MINOR["D"], D_MINOR["Bflat"]]
+    D_MINOR_CHORD_G = [D_MINOR["G"], D_MINOR["E"], D_MINOR["C"]]
```
Comment (gizzmo123456) [inline MelodyGenerator.py]: Mabe just call this CHORD in a dict so its like CHORDS["A"], CHORDS["B"] ...

File: MelodyGenerator.py (hunk 1 above)
Comment (JoachimRayski) [reply MelodyGenerator.py]: I could rename it to D_MINOR_SCALE or something as it's not the D minor chord but all of the pitches in the D minor scale https://en.wikipedia.org/wiki/D_minor i dunno that's what I used for it basically and all the chords use these pitches from that scale

File: MelodyGenerator.py (hunk 2 above)
Comment (JoachimRayski) [reply MelodyGenerator.py]: yeah good idea instead of creating separate variables for them i can just create these lists in the dictionary

**Comment by JoachimRayski** [comment]
@gizzmo123456 yo man did a lil update whatcha think now?

**Pull request merged**